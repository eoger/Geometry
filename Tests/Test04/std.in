C Cercle1 5 5 5
OA
OA N_om Cercle1
OA Nom C*ercle1
OA Nom ObjNonExistant
OA Nom ObjNonExistant1 ObjNonExistant2
OA Nom Cercle1 ObjNonExistant
OA Nom Nom
EXIT