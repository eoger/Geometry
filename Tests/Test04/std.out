OK
ERR
ERR
ERR
ERR
ERR
ERR
ERR